packages/utils/src/format.tsx

```diff
@@ -9,3 +11,14 @@ export function formatEllipsized(text: string, size = 6) {
   }
   return `${text.substring(0, size)}...${text.substring(text.length - size)}`;
 }
+
+/**
+ * Given some number of msats, return a formatted string in the format of
+ * 0.0000000 or 1,234,5678.9000000
+ */
+export function formatMsatsToBtc(msats: MSats): string {
```

Comment:
https://github.com/fedimint/ui/issues/182
Made this issue before I saw this. Changed Issue scope to writing tests for this so the use is more obvious.

Reply:
Yep, would also be good to find all the old code where we're displaying sats or bitcoin and update it to use a set of common utility functions.